astropy/modeling/tests/test_bounding_box.py

```diff
@@ -2476,10 +2459,10 @@ def test__matching_bounding_boxes(self):
         assert bbox.order == 'F'
 
         # Errors
-        with pytest.raises(ValueError) as err:
+        MESSAGE =  ("Attempting to fix input slit_id, but "
+                    "there are no bounding boxes for argument value 2.")
```

Comment:
`.` actually mean something in regex, so either omit or escape to avoid unintentional side effect.
```suggestion
                    "there are no bounding boxes for argument value 2")
```